net/node/node.go

```diff
@@ -28,6 +28,7 @@ import (
 
 const (
 	MaxSyncHeaderReq     = 2                // max concurrent sync header request count
```

Comment:
Similarly, is 2 too small?

Reply:
As we talked about this offline. const 2 is reasonable.